tests/unit_tests/layout/test_layout_array.py

```diff
@@ -232,3 +229,59 @@ def test_altitude_from_corsika_z(layout_center_data_dict, corsika_telescope_data
     )
     with pytest.raises(TypeError):
         layout._altitude_from_corsika_z(5.0, None, "LST-01")
+
+
+def test_include_radius_into_telescope_table(layout_array_instance, telescope_test_file):
+    telescope_table = layout_array_instance.read_telescope_list_file(telescope_test_file)
+    telescope_table_with_radius = layout_array_instance.include_radius_into_telescope_table(
+        telescope_table
+    )
+    values_from_file = np.array([20.190000534057617, -352.4599914550781, 62.29999923706055, 9.6])
+    keys = ["pos_x", "pos_y", "pos_z", "radius"]
+    mst_10_index = telescope_table_with_radius["telescope_name"] == "MST-10"
+    for key_step, _ in enumerate(keys):
+        assert (
+            telescope_table_with_radius[mst_10_index][keys[key_step]].value[0]
+            == values_from_file[key_step]
+        )
+
+
+@pytest.fixture
+def get_manual_corsika_dict_north():
+    return {
+        "corsika_sphere_radius": {
+            "LST": 12.5 * u.m,
+            "MST": 9.15 * u.m,
+            "SCT": 7.15 * u.m,
+            "SST": 3 * u.m,
+        },
+        "corsika_sphere_center": {
+            "LST": 16 * u.m,
+            "MST": 9 * u.m,
+            "SCT": 6.1 * u.m,
+            "SST": 3.25 * u.m,
+        },
+        "corsika_obs_level": 2158 * u.m,
+    }
+
+
+def test_from_corsika_file_to_dict(layout_array_instance, get_manual_corsika_dict_north):
+    corsika_dict = layout_array_instance._from_corsika_file_to_dict()
+    for key, value in corsika_dict.items():
+        if key == "corsika_obs_level":
+            assert value == get_manual_corsika_dict_north["corsika_obs_level"]
+        else:
+            for tel_type, subvalue in value.items():
+                assert subvalue == get_manual_corsika_dict_north[key][tel_type]
+
+
+def test_initialize_corsika_telescope_from_dict(
```

Comment:
I modified this code a bit (and the one in the function above). What's interesting is that when I tried running pytest just of this test it failed cause io_handler wasn't initialised. You didn't see it because when you run all tests one after the other, one of them initialises also io_handler. Therefore, I added `io_handler` in the conftest (made a comment there as well).